Mathlib/Algebra/Order/Field/Basic.lean

```diff
@@ -862,9 +862,9 @@ such that `positivity` successfully recognises `a`. -/
 def evalInv : PositivityExt where eval {u α} zα pα e := do
   let .app (f : Q($α → $α)) (a : Q($α)) ← withReducible (whnf e) | throwError "not ⁻¹"
   let _e_eq : $e =Q $f $a := ⟨⟩
-  let _a ← synthInstanceQ (q(Semifield $α) : Q(Type u))
-  let _a ← synthInstanceQ (q(LinearOrder $α) : Q(Type u))
-  let _a ← synthInstanceQ (q(IsStrictOrderedRing $α) : Q(Prop))
```

Comment:
I understand the comment for getting rid of `: Expr`, but what does removing these type ascriptions do? Is it the same since these are defeq to `Expr` under the hood?